docs/api/Settings.md

```diff
@@ -129,3 +129,32 @@ Store a value of a setting for a given key on a specific document.
 | document<span class="arg-type">[Document](#document) - required</span> | The document on which the setting is set. |
 | key<span class="arg-type">string - required</span>                     | The setting to set.                       |
 | value<span class="arg-type">any - required</span>                      | The value to set it to.                   |
+
+## Get a session variable
+
+```js
+var myVar = Settings.sessionVariable('myVar')
+```
+
+Return the value of a variable which is _not_ persisted when Sketch closes but is persisted when the plugin finishes to run. It is useful when you want to keep a value between plugin's runs.
```

Comment:
this seems like a confusing way to word it? Saying what it's *not* before saying what it *is*

Reply:
Good point!